src/owl/core/owl_tensor_stub.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * OWL - an OCaml numerical library for scientific computing
+ * Copyright (c) 2016-2018 Liang Wang <liang.wang@cl.cam.ac.uk>
+ */
+
+#include "owl_core.h"
+#include <string.h>
```

Comment:
Which function needs to include `string.h`?

Reply:
`memset`, for setting an array to 0.

Reply:
ok, I see, thanks.